lt/disclaimer lt.html

```diff
@@ -0,0 +1,16 @@
+---
+layout: Atsakomybės apribojimas
+ref: Atsakomybės apribojimas
+order: 0
+bgclass: Atsakomybės apribojimas
+
+# translate below, never translate before the first :
```

Comment:
As this line says, do not translate above it, this can not be accepted till this is corrected.